index.js

```diff
@@ -13,7 +13,7 @@ const evalInContext = require('./eval')
 const createMQTTBroker = require('./broker')
 // TODO: send PR to serverless-offline to export this
 const functionHelper = require('serverless-offline/src/functionHelper')
-const createLambdaContext = require('serverless-offline/src/createLambdaContext')
+const createLambdaContext = require('serverless-offline/src/LambdaContext')
```

Comment:
How about something like-

`let createLambdaContext = null;
try {
// Support serverless-offline prior to version x.y.z.
createLambdaContext = require('serverless-offline/src/createLambdaContext')
} catch(e) {
// burgers
}

if (!createLambdaContext) {
try {
// Support serverless-offline after version x.y.z.
createLambdaContext = require('serverless-offline/src/LambdaContext')
} catch(e) {
// fries
}
}`

Reply:
We discussed exactly that.  Long story, long.. Houston doesn't need ssl for serverless iot local anymore so we may not need the fork of serverless-iot-local which then means we can send this fix to tradle as a PR instead of maintaining the fork.. and no, he didn't fix it yet either.  Another ticket for another time.  We'll defer the upgrade of serverless-offline.

Reply:
We still use iot for local serverless-offline testing.  We likely still need this plugin.

Reply:
Is there something in the fork that we need for off-line testing?

Reply:
Yes, we need the plugin for testing.
![image](https://user-images.githubusercontent.com/638810/68022463-bedee880-fc7a-11e9-979b-bcfe168acc1d.png)

Reply:
<img width="632" alt="Screen Shot 2019-11-01 at 7 47 03 AM" src="https://user-images.githubusercontent.com/2048655/68022894-05811280-fc7c-11e9-955a-3de97d5611dd.png">

Reply:
Why would we fork anything?  Keep this plugin.  Problem solved.

Reply:
<img width="329" alt="Screen Shot 2019-11-01 at 8 09 17 AM" src="https://user-images.githubusercontent.com/2048655/68024125-94435e80-fc7f-11e9-842a-2c6bd7d17cb6.png">

Reply:
You're going to need to explain better than word circling (or squaring).

Reply:
Going with the answer you gave me yesterday.